mlflow/pipelines/steps/split.py

```diff
@@ -253,6 +258,36 @@ def _run(self, output_directory):
         self.execution_duration = self.run_end_time - run_start_time
         return self._build_profiles_and_card(train_df, validation_df, test_df)
 
+    def _validate_user_code_output(post_split, train_df, validation_df, test_df):
+        (
+            post_filter_train_df,
+            post_filter_validation_df,
+            post_filter_test_df,
```

Comment:
Should try-catch here too since user may return something not decomposable to a 3-tuple.